src/components/calcite-notice/calcite-notice.tsx

```diff
@@ -91,31 +92,38 @@ export class CalciteNotice {
   }
 
   render(): VNode {
-    const dir = getElementDir(this.el);
+    const { el } = this;
+    const dir = getElementDir(el);
     const closeButton = (
       <button
         aria-label={this.intlClose}
-        class="notice-close"
+        class={CSS.close}
         onClick={this.close}
         ref={() => this.closeButton}
       >
         <calcite-icon icon="x" scale={this.scale === "s" ? "s" : "m"} />
       </button>
     );
 
+    const hasAction = getSlotted(el, SLOTS.actionEnd);
```

Comment:
nitpick: Should we call `hasActionEnd` incase we have another action start?

Reply:
We sure can. 👍🏼